02-express-tutorial/answers/final/012-router-app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const app = express()
+// const people  = require('./routes/people')
+const people  = require('./routes/people_2')
+const auth = require('./routes/login')
```

Comment:
Probably just a copy/paste error, but now these files are up a directory and this doesn't work; and, the `login` file is named `auth`.

Reply:
Agreed! Poor attention to detail on my part. This has been corrected